Users/abehenderson/CSX/Recursion/repeater.js

```diff
@@ -0,0 +1,31 @@
+// Attempt #2 (CSX Solution)
+
+function repeater(char) {
+  if (char.length === 5) return char;
+    return repeater(char + char[0]);
+}
+
+console.log(repeater('g')); // logs: 'ggggg'
+console.log(repeater('j')); // logs: 'jjjjj'
+
+//-------------------------------------------------------------------------------------
+
+// // Attempt #1 (I used a counter in global memory because that's exactly how CSX taught it in the video called 'Recursion: Repeater')
+
+// let counter = 0;
+
+// function repeater(char) {
+//   counter++;  
+//   if (counter >= 5) {
+//     counter = 0;
+//     return char;
+//   } else {
+//     return char + repeater(char);
```

Comment:
On the first call, the return value is `g + repeater(g)`... does this attempt even work? I'm not sure just looking at it. I can see a place it might break down.

Reply:
Yes it worked for me.  But, are you saying that `return repeater(char + char[0];` is a better syntax to use?